src/encoded/static/components/submission/submission-fields.js

```diff
@@ -65,6 +65,11 @@ export default class BuildField extends React.Component{
                     <input id="floatNumber" type="number" inputMode="latin" {...inputProps} />
                 </div>
             );
+            case 'boolean' : return (
```

Comment:
IN PROGRESS -> Adding checkbox for boolean fields. May need to have custom onChange or "checked" prop on it.